### PR TITLE
made polished, user friendly so it shouldn't be hard to use anymore

### PR DESCRIPTION
**Pull request by Vistril**
pls test

### CODE REVIEW

**Comment by ledlamp** [comment]
as if it was hard to use >:(

File: manager.js [hunk 1]
```diff
@@ -9,8 +9,20 @@ String.prototype.capitalize = function() {
...
+//constantn'ts this is up here for scope reasons don't @ me
+var StatusCycleCount = 0;
+var OnlineServers = [];
+
+//constants
 var MAX_IDLE_MINUTES = 60;
 var CMD_PREFIX = '%';
+const DEFAULT_STATUS = `${CMD_PREFIX}help`
+const STATUS_CYCLE = [
+	"Servers running: " + OnlineServers.join(", "),
```
Comment (ledlamp) [inline manager.js:21]: `Object.keys(servers).filter(name => servers[name].process)`

**Comment by ledlamp** [comment]
i just got an idea tho

**Comment by Vistril** [comment]
Kinda was but I'm not in this server anymore so I don't care

**Pull request closed without merging**